aayushisingh11/CountZeros.c

```diff
@@ -0,0 +1,23 @@
+#include <stdio.h>
+
+void main()
+{
+    int arr[50], n, i, positive = 0, negative = 0, zero = 0;
+
+    printf("Enter number of elements: ");
+    scanf("%d", &n);
+
+    printf("Enter array elements-\n");
+    for (i = 0; i < n; i++)
+        scanf("%d", &arr[i]);
+
+    for (i = 0; i < n; i++)
+    {
+        if (arr[i] > 0)
+            positive++;
+        else if (arr[i] < 0)
+            negative++;
+        else
+            zero++;
```

Comment:
Looks okay. You can now just put in a function `int CountZeros(int arr[])` that returns the `zeros`.

Reply:
Okay!

Reply:
Now I have updated and added function CountZeros[ ]
https://github.com/aayushisingh11/micropilot-entry-challenge/blob/main/aayushisingh11/CountZeros.c